charts/ceph-csi-cephfs/README.md

```diff
@@ -133,6 +133,7 @@ charts and their default values.
 | `driverName`                                   | Name of the csi-driver                                                                                                                               | `cephfs.csi.ceph.com`                              |
 | `configMapName`                                | Name of the configmap which contains cluster configuration                                                                                           | `ceph-csi-config`                                  |
 | `externallyManagedConfigmap`                   | Specifies the use of an externally provided configmap                                                                                                | `false`                                            |
+| `cephConfigMapName`                            | Name of the configmap which contains ceph.conf configuration                                                                                           | `ceph-config`                                  |
```

Comment:
can you rename this variable to `cephConfConfigMapName`?

Reply:
The reason I picked this name is that it can include other Ceph config files in the future beyond ceph.conf plus it keeps the variable name short. I can change it though.

Reply:
It gets complicated. we can have more configmaps its not a problem.

Reply:
Understood, I just pushed a new commit to rename the variable. Thanks for the feedback.